root.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
 <head>
     <meta charset="utf-8">
-    <meta http-equiv="Content-Security-Policy" content="script-src 'self' cdn.segment.com/analytics.js/">
```

Comment:
Is there a reason for removing this?

Reply:
Yes - react HMR requires 'eval' to be enabled. The CSP was removed from there but it is added by webpack now, albeit a bit lower in the head section usually. For non-production (and if IIRC, any non-dev-server targets) the CSP will be the same text as previously. Within the dev-server it will have 'eval' added.

Reply:
Aah i missed that. Thanks LGTM